tools/oled_preview.py

```
@@ -86,17 +86,27 @@ class Lang:
     def __init__(self, xlsx: str, named: dict):
         from openpyxl import load_workbook
         wb = load_workbook(xlsx, data_only=True, read_only=True)
-        self.sh = wb['key_lut']; self.named = named
+        ws = wb['key_lut']; self.named = named
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)